jobs/e2e_node/containerd/containerd-release-1.5/image-config.yaml

```diff
@@ -4,6 +4,6 @@ images:
     project: ubuntu-os-gke-cloud
```

Comment:
Let's change this to image_family: `ubuntu-os-cloud`
image_family: `ubuntu-2204-lts`

Reply:
edit: as discussed on slack ([thread](https://kubernetes.slack.com/archives/C0BP8PW9G/p1668164946763269)), let's keep release-1.5 on non cgroupv2 since containerd 1.5 does not have the systemd cgroup configuration option.

Reply:
The image for [containerd-main/cgroupv1 ](https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/containerd/containerd-main/cgroupv1/image-config-cgroupv1.yaml) is pointing to cos-stable. Should we revert it back? or the [`CONTAINERD_COS_CGROUP_MODE: 'v1'`](https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/containerd/containerd-main/cgroupv1/env-cgroupv1#L9) is enough ?

Reply:
@bobbypage should we continue using cos-89-lts for containerd-1.5 job? as containerd 1.5 doesn't have both systemd and cgroup driver settings?

Reply:
Sounds good, let's keep 1.5 jobs on older COS since it's older release and needs CRI v1alpha1